_pages/plugins/ciliaq.md

```diff
@@ -0,0 +1,30 @@
+---
+mediawiki: CiliaQ
+title: CiliaQ
+description: A set of three ImageJ plugins to quantify ciliary shape, length, and fluorescence in 2D, 3D, and 4D images of cilia.
+icon: https://github.com/hansenjn/CiliaQ/blob/master/Webfiles/20200618%20CiliaQ%20Logo%20Small.png
+categories: [Automation, 3D, Analysis, Particle Analysis, Segmentation, Object Detection, Colocalization, Tracking, Cilia]
+doi: 10.1140/epje/s10189-021-00031-y
+source-url: https://github.com/hansenjn/CiliaQ/
+dev-status: Stable
+support-status: Active
+team-founder: '@hansenjn'
+team-maintainer: '@hansenjn'
+---
+
+CiliaQ refers to a set of three ImageJ plugins to quantify ciliary shape, length, and fluorescence in 2D, 3D, and 4D images of cilia,
+acquired with a fluorescence / confocal microscope. 
+
+The software is published with the following publication:
+Jan N. Hansen, Sebastian Rassmann, Birthe Stueven, Nathalie Jurisch-Yaksi, Dagmar Wachten. 
+*CiliaQ: a simple, open-source software for automated quantification of ciliary morphology and fluorescence in 2D, 3D, and 4D images*. 
+Eur. Phys. J. E 44, 18 (2021). https://doi.org/10.1140/epje/s10189-021-00031-y
```

Comment:
Please use the `{% include citatation}` mechanism to cite publications. See https://imagej.net/editing/citations